examples/examples.mk

```diff
@@ -140,6 +140,18 @@ o/$(MODE)/examples/hello.com.dbg:				\
 		$(APE_NO_MODIFY_SELF)
 	@$(APELINK)
 
+o/$(MODE)/examples/picol.o:					\
```

Comment:
I added this rather hastily... is there a logical place this should have been placed (alpha order, etc?).

Reply:
I sent the copyright assignment.